py-notebook/README.md

```diff
@@ -0,0 +1,49 @@
+# A Jupyter Lab &amp; Jupyter Notebook widget for Neo
+
+## Setup
+
+To use Neo in Jupyter Lab or Jupyter Notebook, install a local Jupyter extension for neo from the command line:
+
+1. In a terminal, navigate to the included jupyter extension folder:
+
+`$ cd neo-confusion-jupyter`
+
+2. Locally install the neo jupyter extension package to Python:
+
+`$ pip install -e .`
```

Comment:
When I call this (from the `jupyter-widget` folder), I get the following `webpack` error:

```
ERROR in main
    Module not found: Error: Can't resolve './lib/embed.js' in '/Users/.../ml-hierarchical-confusion-matrix/py-notebook/jupyter-widget/js'
```